src/gui/macosshiftclickhandler.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2025  Luke Memet <lukemmtt@users.noreply.github.com>
```

Comment:
Having a non working email is meaningless. You might just remove it.
Or put your github username instead: `Luke Memet (lukemmtt)`